neural_speed/__init__.py

```diff
@@ -402,6 +403,12 @@ def generate(self,
     def is_token_end(self):
         return self.model.is_token_end()
 
+    def reset_kv_cache(self):
+        if self.model is None:
+            return
```

Comment:
if self.model is None:
  may be we could throw a warning log like:
  The model is not inited yet, then return

Reply:
fixed in [47c3f09](https://github.com/intel/neural-speed/pull/227/commits/47c3f0967eeaff42b6b31286f03dcba3af721743)